lib/kovid/request.rb

```diff
@@ -117,10 +117,17 @@ def state(state)
         puts SERVER_DOWN
       end
 
-      def states(list)
-        array = fetch_states(list)
+      def states(states)
+        compared_states = fetch_compared_states(states)
```

Comment:
Updated argument name and variable declaration for the sake of clarity. In general I think it will be best if we avoid names like `array`, as it is less clear than alternatives.